csharp/src/Ice/Util.cs

```diff
@@ -106,6 +107,16 @@ public object Clone()
         /// The list of TypeId namespaces. Default is Ice.TypeId.
         /// </summary>
         public string[] typeIdNamespaces = { "Ice.TypeId" };
+
+        /// <summary>
+        /// The callback for configure the server-side ssl transport.
+        /// </summary>
+        public IceSSL.SslServerAuthenticationOptionsCallback sslServerAuthenticationOptionsCallback;
+
+        /// <summary>
+        /// The <see cref="SslClientAuthenticationOptions"/> used bty the client-side ssl transport.
+        /// </summary>
+        public SslClientAuthenticationOptions sslClientAuthenticationOptionsCallback;
```

Comment:
This will allow configuring SSL transport using the C# platforms API.

Reply:
I would remove "callback" from the field name here.